task10/index.js

```diff
@@ -0,0 +1,63 @@
+const div = document.getElementById("js-parent");
+
+const loading = () => {
+    const loadingPlace = document.createElement("div");
+    const gif = document.createElement('img');
+    loadingPlace.id = "loadingPlace";
+    gif.src = "loading-circle.gif";
+    div.appendChild(loadingPlace).appendChild(gif);
+};
+
+const removeLoading = () => {
+    document.getElementById("loadingPlace").remove();
+};
+
+const getData = () => {
+    const obj = [
+        {to: "bookmark.html", img: "1.png", alt:"画像1", text: "ブックマーク"}, 
+        {to: "message.html", img: "2.png", alt:"画像2", text: "メッセージ"}
+    ]
+    
+    return new Promise((resolve,reject) => {
+        setTimeout(() => {
+            resolve(obj);
+            reject("error!");
+        },2000);
```

Comment:
The way it's written, both resolve and reject will be called.
In this case, I thought it would be more natural to write, "If obj is ture, run resolve, if false, run reject.
(I struggled with it too.😂)
IMO
```suggestion
           setTimeout(() => {
        if(obj){
            resolve(obj);
        }else{
            reject("error!");
        }
    }, 2000);
```

Reply:
You're right!
I fixed it using if, I'm not very good at using if so I'll learn more. Thank you very much.